castai/resource_node_configuration.go

```diff
@@ -286,6 +287,12 @@ func resourceNodeConfiguration() *schema.Resource {
 								},
 							},
 						},
+						"node_group_arn": {
+							Type:             schema.TypeString,
+							Optional:         true,
+							Description:      "Cluster's node group ARN used for CAST provisioned node pools",
```

Comment:
Is this only used for hibernate or for all CAST pools? Does it apply only to EKS?  Some additional context in the description would be useful.

Reply:
EKS only, will update description.